README.md

```diff
@@ -45,56 +46,54 @@ events from the device like alerts, removal/insertion events. It then publishes
 | PSME | |  PSME  | | Redfish |
 +------+ +--------+ +---------+
 ```
-The Edgecore Device Manager could manage a wide range of devices (ex: Edgecore XGS-PON/GPON vOLTs, ONL/SONIC Switches, and OpenBMC device).
-Before you use the Device Manager, you have to install the PSME software to the device.
-More detailed information can be found at [DM-Redfish-PSME](https://github.com/opencomputeproject/DM-Redfish-PSME) and [DM-Redfish-OpenBMC](https://github.com/opencomputeproject/DM-Redfish-OpenBMC).
 
-# Preparation
+The device can implement the Redfish Service on an out-of-band platform entity (e.g. BMC) or on an in-band entity (e.g software agent).
+
+The Redfish Service should be conformant to the OCP Baseline Profile.
+The [OCP Baseline profile](https://github.com/opencomputeproject/HWMgmt-OCP-Profiles) prescribes the required [Redfish interface](http://dmtf.org/redfish) support.
+
+The Linux Foundation OpenBMC repository contains source for a BMC firmware image with is conformant to the OCP Baseline profile.
```

Comment:
Link to OpenBMC github project would be nice.

Reply:
Added reference.